index.js

```diff
@@ -356,6 +356,19 @@
     ($.Undefined  = type0('Undefined'))
   ];
 
+  // Enum :: [Any] -> Type
+  $.Enum = function(constants) {
+    return {
+      type: 'sanctuary-def/Enum',
+      test: function(x) {
+        return $.Array(Any).test(constants) && constants.indexOf(x) !== -1;
```

Comment:
Ah wait, I was confusing that with `_show`, which is a curried function of arity 2. Oops.